qemu/tests/cpu_flag_intel_pt.py

```diff
@@ -0,0 +1,83 @@
+import logging
+
+from avocado.utils import cpu, process
+from virttest import error_context, env_process
+from qemu.tests.x86_cpu_model import check_flags
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test cpu flag intel-pt.
+    1) Check if current flags are in the supported lists, if no, cancel test
+    2) Otherwise, set pt_model = 1 on host at first
+    3) For q35
+       3.1) boot guest with cpu model 'host' without intel-pt
+       3.2) boot guest with cpu model 'Icelake-Server' with intel-pt
+    4) For pc
+       4.1) boot guest with intel-pt and min-level=0x14
```

Comment:
Would you please have a try with '-cpu host -M pc'? Besides,  maybe step 3 and step 4 can be implemented in cfg?